ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/transaction/ApiTransactionService.scala

```diff
@@ -105,7 +105,7 @@ class ApiTransactionService private (
       .split(request.eventId.unwrap)
       .fold(
         Future.failed[Transaction](
-          Status.INVALID_ARGUMENT
+          Status.NOT_FOUND
```

Comment:
This should remain as `INVALID_ARGUMENT` because the eventID didn't match the expected format.

Reply:
that's hard to implement. the daml-on-x cannot validate eventId form, so it's not clear how to write the test for this. lets follow up on fixing this!

Reply:
After discussing offline with @dajmaki, we agreed that `NOT_FOUND` is the correct error status, because the ledger api never specifies the format of the event id. Returning `INVALID_ARGUMENT` without telling users how to make it valid (i.e. by way of specifying the format) is not a nice thing to do :)